nodebuilder/share/cmds/share.go

```diff
@@ -190,7 +177,13 @@ var getEDS = &cobra.Command{
 			return err
 		}
 
-		shares, err := client.Share.GetEDS(cmd.Context(), &root)
-		return printOutput(shares, err, nil)
+		shares, err := rpcClient.Share.GetEDS(cmd.Context(), &root)
+		return util.PrintOutput(shares, err, nil)
 	},
 }
+
+func parseJSON(param string) (json.RawMessage, error) {
```

Comment:
imo it's fine to just move this up vs create the parseJSON function, makes the above command more readable, making the function either should live in util (i'm sure it'll happen again) or we just be verbose above